release_notes/49-release-notes.adoc

```diff
@@ -0,0 +1,478 @@
+
+For more information, see xref:../configuration/configure-api-token.adoc#configure-api-token[Configuring API tokens].
+
+//ROX-29959, ROX-29070
+[id="declarative-config-m2m_{context}"]
+=== Declarative configuration for M2M access configuration
+
+You can now declaratively configure M2M OIDC authentication. To configure M2M authentication resources, you create YAML files that contain configuration information. These files are used to create a config map or secret that is added to Central by using a mount point during installation of the {product-title-short} Central resource. 
+
+For an example of setting up declarative configuration for short-lived OIDC token usage, see xref:../configuration/declarative-configuration-using.adoc#declarative-config-example-short-lived-token[Declarative configuration short-lived token example].
+
+//Security
+
+//ROX-20262
+[id="certificate-rotation-operator-managed-clusters_{context}"]
+=== Automatic certificate authority (CA) rotation for Operator-managed clusters
+
+With this update, the Central CA certificate, which has a validity of 5 years, will be automatically rotated for Operator-managed Central and secured clusters. Previously, in release 4.7, {product-title-short} automatically rotated the 1-year service certificates, and now rotation is provided for the CA certificates. This enhancement simplifies the management of large cluster installations, as manually upgrading certificates can be a significant task. 
+
+[NOTE]
+====
+Automatic certificate rotation remains partially supported for Helm-installed secured clusters. These clusters can connect to Central with the rotated CA but their certificates remain signed by the older CA.
+====
+
+For more information, see xref:../configuration/reissue-internal-certificates.adoc#reissue-internal-certificates[Reissuing internal certificates].
+
+//ROX-27238
+[id="crs-expiration-time_{context}"]
+=== Configurable expiration time for cluster registration secrets
+
+With this release, the expiration time for cluster registration secrets (CRSes) is changed from a year to a default value of 1 hour. Additionally, the time period can be configured by using the `--valid-until` or `--valid-for` flags with the `roxctl central crs` command. 
+
+For more information, see xref:../cli/command-reference/roxctl-central.adoc#roxctl-central-crs_roxctl-central[`roxctl central crs` command].
+
+
+//ROX-30654
+[id="smtp-ehlo-helo-hostname_{context}"]
+=== SMTP EHLO/HELO hostname field supported in the {product-title-short} email notifier
+
+{product-title-short} SMTP notifiers now support configuring the EHLO/HELO hostname. This capability allows for better compatibility with strict mail relay servers in secured environments.
```

Comment:
RH Supplementary > Release notes > Tenses in release notes
avoid use of "now"

Suggestion:
"With the 4.9 release, SMTP notifiers....."